src/material-experimental/mdc-list/list.scss

```diff
@@ -63,6 +63,20 @@
   }
 }
 
+.mat-mdc-list-avatar {
+  // MDC's styles don't specify this, but they probably should. It gives a nicer experience when the
+  // image is not 1:1 aspect ratio.
+  object-fit: cover;
+}
+
+// MDC expects that the list items are always `<li>`, since we actually use `<button>` in some
+// cases, we need to make sure it expands to fill the available width.
```

Comment:
Does their CSS specifically have `li`, or does `li` just naively have `width: 100%`? Where would the `box-sizing` normally come from?

Reply:
`li` naturally fills the width of the `ul`, using `width` and `box-sizing` here achieves the same thing if we're not using `ul` and `li`